internal/bminventory/inventory.go

```diff
@@ -1499,6 +1521,15 @@ func (b *bareMetalInventory) updateClusterData(ctx context.Context, cluster *com
 	if params.ClusterUpdateParams.NoProxy != nil {
 		updates["no_proxy"] = swag.StringValue(params.ClusterUpdateParams.NoProxy)
 	}
+	if params.ClusterUpdateParams.NtpSource != nil {
```

Comment:
Not necessarily for this PR.
Seems that RegisterCluster and updateClusterData have a lot of common code.
Consider extracting it to a common function or let register use update.